cardinal/world.go

```diff
@@ -285,7 +286,14 @@ func (w *World) Tick(ctx context.Context) error {
 	return w.implWorld.Tick(ctx)
 }
 
-func (w *World) Init(fn func(WorldContext)) {
+func (w *World) RunInContextOfWorldContext(fn func(WorldContext)) {
```

Comment:
I'd say remove this entirely and use `cardinal.TestingWorldToWorldContext` in the tests that need it.